### ISSUE TITLE
Items displayed using the Internet Archive Book Reader do not expose the full text to someone using a screenreader (Am I missing something?)

### ISSUE THREAD

**Issue by wrandtkeflvc** [FLVC/flvc#88]
Items displayed in Islandora using the  Internet Archive Book Reader do not expose the full text to someone using a screenreader.  Islandora stores the full text, and uses it for search, but the only way someone can read the document seems to be to look at the jpeg displayed on the screen in the page flipper view.  (Someone can go page by page and download a PDF of each page one at a time, but that's tedious.)

When I look at other implementations of Internet Archive Book Reader, I see some are providing a way to get full text:
- Boston College allows download of a PDF for all Islandora objects.  Then someone can have that PDF and copy paste or display the embedded text in the PDF.
- Internet Archive allows download of EPub, text files of the book, etc.  It has lots of download options to where someone probably can get the full text to be able to read the book.

This matters for accessibility.  FLVC should look at settings to see whether it's possible to do something more to expose full text to a screen reader, and should bring this up to the Islandora community.
